public/js/main.js

```diff
@@ -3,46 +3,47 @@ function callAjaxForCreateMenu($url, $categoryId) {
         headers: {
             'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
         }
-        });
-        $.ajax({
-            type: "GET",
-            url: $url,
-            data: {
-                category_id: $categoryId
-            },
-            success: function(data) {
-                data['data'].forEach(function(foodElement) {
-                    //create food select
-                    let $foodOption = $("<option>", {"text": foodElement['name'], "value": foodElement['id']});
-                    $("#foodselect").append($foodOption);
-                })
-            }
-        });
+    });
+    $.ajax({
+        type: "GET",
+        url: $url,
+        data: {
+            category_id: $categoryId
+        },
+        success: function (data) {
+            data['data'].forEach(function (foodElement) {
+                //create food select
+                let $foodOption = $("<option>", {"text": foodElement['name'], "value": foodElement['id']});
+                $("#foodselect").append($foodOption);
```

Comment:
Change
```"<option>"```
To:

```'<option>'```
Change:
```$("#foodselect")```
To:
```$('#select-food')```
And anywhere.... 
In html, javascript naming ID will be using like that.